### ISSUE TITLE
CIEL v2022-12-21 release

### ISSUE THREAD

**Issue by bmamlin** [OpenConceptLab/ocl_issues#1447]
- [x] Convert to JSON
- [x] Create any new sources ([OMOP-Extension](https://app.openconceptlab.org/#/orgs/OHDSI/sources/OMOP-Extension/) and [ANC-DAK](https://app.staging.openconceptlab.org/#/orgs/WHO/sources/ANC-DAK/))
- [x] Upload zip file to [liquibaserunner](https://github.com/openmrs/openmrs-contrib-liquibaserunner)

### Staging
- [x] Create diff
- [x] Retire any deleted mappings
- [x] Bulk import
- [x] Validate
- [x] Release

### Production
- [x] Create diff
- [x] Retire any deleted mappings
- [x] Bulk import
- [x] Validate
- [x] Release

**Comment by bmamlin** [comment]
@snyaggarwal,

An exception occurred while importing concepts on production: 😕 
> TASK `6b96a7c8-7fa6-4dd6-a11c-6db511caa0d4-ocladmin~priority`
> **deadlock detected**
> DETAIL:  Process 30993 waits for ShareLock on transaction 22361343; blocked by process 1882.
> Process 1882 waits for ShareLock on transaction 22361339; blocked by process 30993.
> HINT:  See server log for query details.
> CONTEXT:  while updating tuple (5131,10) in relation "concepts"

**Comment by bmamlin** [comment]
The following 69 concepts ended up with duplicate mappings on production:
* [161602](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/161602/) - /orgs/OpenMRS/sources/org.openmrs.module.emrapi/concepts/Non-Coded+Diagnosis/
* [160051](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160051/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/CURRENT+MULTI-DRUG+RESISTANT+TUBERCULOSIS+TREATMENT+TYPE/
* [160038](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160038/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/MULTI-DRUG+RESISTANT+TUBERCULOSIS+TREATMENT+OUTCOME/
* [160037](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160037/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/STILL+ON+TREATMENT/
* [160036](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160036/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/PATIENT+TRANSFERRED+OUT/
* [160035](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160035/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TREATMENT+COMPLETE/
* [160030](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160030/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TREATMENT+AFTER+FAILURE+OF+RE-TREATMENT+MDR-TB+PATIENT/
* [160027](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160027/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/SCANNED+LAB+REPORT/
* [160024](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160024/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/HOSPITALIZATION+WORKFLOW/
* [160021](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160021/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+DRUGS/
* [159983](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159983/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+DRUG+SENSITIVITY+TEST+CONSTRUCT/
* [159982](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159982/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+CULTURE+RESULT/
* [159981](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159981/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+CULTURE+CONSTRUCT/
* [159978](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159978/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/PREVIOUSLY+TREATED+WITH+FIRST+LINE+DRUGS+ONLY/
* [159971](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159971/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/WAITING+FOR+TEST+RESULTS/
* [159968](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159968/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+SPECIMEN+ID/
* [159967](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159967/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TYPE+OF+ORGANISM+NON-CODED/
* [159965](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159965/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+TEST+START+DATE/
* [159964](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159964/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+TEST+RESULT+DATE/
* [159963](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159963/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+TEST+DATE+RECEIVED/
* [159962](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159962/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+TEST+DATE+ORDERED/
* [159961](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159961/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+SMEAR+MICROSCOPY+METHOD/
* [159960](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159960/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+SMEAR+MICROSCOPY+CONSTRUCT/
* [159959](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159959/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+SAMPLE+SOURCE/
* [159956](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159956/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/RESISTANT+TO+TUBERCULOSIS+DRUG/
* [159955](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159955/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/COLONIES+IN+CONTROL/
* [159953](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159953/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+CULTURE+METHOD/
* [159947](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159947/) - /orgs/OpenMRS/sources/org.openmrs.module.emrapi/concepts/Diagnosis+Concept+Set/
* [159946](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159946/) - /orgs/OpenMRS/sources/org.openmrs.module.emrapi/concepts/Diagnosis+Order/
* [159635](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159635/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TELEPHONE+NUMBER/
* [159395](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159395/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/CLINICIAN+NOTES/
* [159346](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159346/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/EXTENSIVE+DRUG+RESISTANT+TUBERCULOSIS/
* [160039](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160039/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/SUSPECTED+MULTI-DRUG+TUBERCULOSIS/
* [160032](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160032/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TREATMENT+AFTER+FAILURE+OF+FIRST+TREATMENT+MDR-TB+PATIENT/
* [160015](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160015/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/MULTI-DRUG+RESISTANT+TUBERCULOSIS+CULTURE+STATUS/
* [160001](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160001/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/OTHER+MYCOBACTERIA+NON-CODED/
* [159991](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159991/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/COINFECTED+AND+ON+ANTIRETROVIRALS/
* [159990](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159990/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/CATEGORY+4+TUBERCULOSIS+CLASSIFICATION+ACCORDING+TO+RESULT+OF+PREVIOUS+TREATMENT/
* [159986](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159986/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/DAYS+TO+POSITIVITY/
* [159979](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159979/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/PREVIOUSLY+TREATED+WITH+SECOND+LINE+DRUGS/
* [159976](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159976/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/CATEGORY+4+TUBERCULOSIS+CLASSIFICATION+ACCORDING+TO+PREVIOUS+DRUG+USE/
* [159972](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159972/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/DST+CONTAMINATED/
* [159970](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159970/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+SPECIMEN+COMMENTS/
* [159969](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159969/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/APPEARANCE+OF+SPECIMEN/
* [159966](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159966/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TYPE+OF+ORGANISM/
* [159958](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159958/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/SUSCEPTIBLE+TO+TUBERCULOSIS+DRUG/
* [159957](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159957/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/INTERMEDIATE+TO+TUBERCULOSIS+DRUG/
* [159954](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159954/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+DRUG+SENSITIVITY+TEST+METHOD/
* [159951](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159951/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/SPUTUM+COLLECTION+DATE/
* [159427](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159427/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/RESULT+OF+HIV+TEST/
* [159394](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159394/) - /orgs/OpenMRS/sources/org.openmrs.module.emrapi/concepts/Diagnosis+Certainty/
* [159345](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159345/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/MULTI-DRUG+RESISTANT+TUBERCULOSIS/
* [126224](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/126224/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/UNSATISFACTORY+SAMPLE/
* [81457](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/81457/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/P-AMINOSALICYLIC+ACID/
* [5242](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/5242/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/RESPIRATORY+RATE/
* [5042](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/5042/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/EXTRA-PULMONARY+TUBERCULOSIS/
* [5497](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/5497/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/CD4+COUNT/
* [5096](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/5096/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/RETURN+VISIT+DATE/
* [5085](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/5085/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/SYSTOLIC+BLOOD+PRESSURE/
* [1364](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/1364/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/STRONGLY+POSITIVE/
* [1362](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/1362/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/WEAKLY+POSITIVE/
* [1284](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/1284/) - /orgs/OpenMRS/sources/org.openmrs.module.emrapi/concepts/Coded+Diagnosis/
* [1363](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/1363/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/MODERATELY+POSITIVE/
* [1269](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/1269/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/REASON+TUBERCULOSIS+TREATMENT+CHANGED+OR+STOPPED/
* [1252](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/1252/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/REASON+ANTIRETROVIRALS+CHANGED+OR+STOPPED/
* [1085](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/1085/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/ANTIRETROVIRAL+DRUGS/
* [307](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/307/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TUBERCULOSIS+SMEAR+RESULT/
* [450](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/450/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/AMOXICILLIN+AND+CLAVULANIC+ACID/
* [42](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/42/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/PULMONARY+TUBERCULOSIS/

**Comment by snyaggarwal** [comment]
@bmamlin These were retired and HEAD export was retriggered.

**Comment by bmamlin** [comment]
Thanks @snyaggarwal.

On Staging, 12 concepts ended up with duplicate mappings following import of CIEL v2022-12-21:

These 11 have duplicate NARROWER-THAN mappings:
* [165897](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165897/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FJB63.Y/
* [165887](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165887/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCA20.Z/
* [165886](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165886/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FAB0Z/
* [165885](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165885/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2F1A2Z/
* [165884](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165884/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2F8E47/
* [165883](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165883/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FBC42.Y/
* [165869](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165869/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCA42.Y/
* [165868](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165868/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCA4Z/
* [165867](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165867/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCB00/
* [165866](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165866/) - /orgs/WHO/sources/ICD-11-WHO/concepts/QC42%2FRA01.1/
* [165608](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165608/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCA40.1Y/

This concept has a duplicate SAME-AS mapping:
* [160009](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160009/ - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/DIRECT%2FINDIRECT/

Any idea why these are cropping up? Am I doing something wrong? Let me know if/when you want me to retire the duplicates of these mappings. Once they're deleted, I'll need to generate a new export of HEAD for validation.

**Comment by bmamlin** [comment]
@snyaggarwal,

I still see four duplicate mappings for HEAD on prod:

* [160036](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160036/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/PATIENT+TRANSFERRED+OUT/
* [159635](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159635/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TELEPHONE+NUMBER/
* [159395](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159395/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/CLINICIAN+NOTES/
* [159346](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159346/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/EXTENSIVE+DRUG+RESISTANT+TUBERCULOSIS/

Are these regenerating? Or do you think you might've missed 4 of 69? 😄  I'm happy to retire these and regenerate an export of HEAD, but want you to know in case it helps you troubleshoot & squash bugs. Let me know when I can retire them & regenerate an export of HEAD.

**Comment by snyaggarwal** [comment]
> @snyaggarwal,
> 
> I still see four duplicate mappings for HEAD on prod:
> 
> * [160036](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160036/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/PATIENT+TRANSFERRED+OUT/
> * [159635](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159635/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/TELEPHONE+NUMBER/
> * [159395](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159395/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/CLINICIAN+NOTES/
> * [159346](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/159346/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/EXTENSIVE+DRUG+RESISTANT+TUBERCULOSIS/
> 
> Are these regenerating? Or do you think you might've missed 4 of 69? 😄 I'm happy to retire these and regenerate an export of HEAD, but want you to know in case it helps you troubleshoot & squash bugs. Let me know when I can retire them & regenerate an export of HEAD.

@bmamlin I missed these 4. Retired now and regenerating HEAD.

**Comment by snyaggarwal** [comment]
> Thanks @snyaggarwal.
> 
> On Staging, 12 concepts ended up with duplicate mappings following import of CIEL v2022-12-21:
> 
> These 11 have duplicate NARROWER-THAN mappings:
> 
> * [165897](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165897/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FJB63.Y/
> * [165887](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165887/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCA20.Z/
> * [165886](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165886/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FAB0Z/
> * [165885](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165885/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2F1A2Z/
> * [165884](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165884/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2F8E47/
> * [165883](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165883/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FBC42.Y/
> * [165869](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165869/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCA42.Y/
> * [165868](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165868/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCA4Z/
> * [165867](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165867/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCB00/
> * [165866](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165866/) - /orgs/WHO/sources/ICD-11-WHO/concepts/QC42%2FRA01.1/
> * [165608](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/165608/) - /orgs/WHO/sources/ICD-11-WHO/concepts/RA01.1%2FCA40.1Y/
> 
> This concept has a duplicate SAME-AS mapping:
> 
> * [160009](https://app.staging.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/160009/) - /orgs/OpenMRS/sources/org.openmrs.module.mdrtb/concepts/DIRECT%2FINDIRECT/
> 
> Any idea why these are cropping up? Am I doing something wrong? Let me know if/when you want me to retire the duplicates of these mappings. Once they're deleted, I'll need to generate a new export of HEAD for validation.

@bmamlin I retired these 12 mappings and triggered the HEAD export.
I don't think you are doing anything wrong, it's the data these env have. The duplicates are coming with each import because some of the old version data is not correct, I thought I fixed that, but clearly not!
I am looking into this.

**Comment by snyaggarwal** [comment]
@bmamlin I have deployed the fix on Staging and reimported one of the duplicate mappings, and it worked fine (no duplicates). Will trigger HEAD export now

**Comment by bmamlin** [comment]
Thanks @snyaggarwal. I've validated & released CIEL v2022-12-21 on staging & production (I deleted the version @askanter released on production and recreated it, since the version he released hadn't been validated and had duplicate mappings in it).
